tools/scenario-player/scenario_player/utils.py

```diff
@@ -268,7 +268,7 @@ def get_udc_and_token(runner) -> Tuple[Optional[ContractProxy], Optional[Contrac
 
     udc_address = udc_config.get('address')
     if udc_address is None:
-        contracts = get_contracts_deployed(chain_id=runner.chain_id, services=True)
+        contracts = get_contracts_deployment_info(chain_id=runner.chain_id)
```

Comment:
We should set a contracts version here.

At least this is done in https://github.com/raiden-network/raiden/blob/master/tools/scenario-player/scenario_player/tasks/blockchain.py#L124 and we should be consistent.

Reply:
Maybe `contracts_version` should not be optional.  https://github.com/raiden-network/raiden-contracts/issues/801

Reply:
Is the contracts version somehow configurable in the SP? Or always development?

Reply:
See https://github.com/raiden-network/raiden/pull/3659#discussion_r268117204

Reply:
Changed it to dev version now, but I agree that making it configurable is a good idea

Reply:
@palango 

> See #3659 (comment)

This links to this very thread :smiley: